app/main.py

```diff
@@ -2,0 +1,40 @@
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
```

Comment:
Appending `self` to the `Animal.alive` list in the constructor is correct for tracking alive animals. However, ensure that the `check_health` method is called appropriately to remove dead animals from this list.